pkg/rotator/rotator_test.go

```diff
@@ -219,7 +219,7 @@ func TestReconcile(t *testing.T) {
 	err = c.Create(ctx, wh)
 	g.Expect(err).NotTo(gomega.HaveOccurred(), "creating webhookConfig")
 
-	stop, wg := StartTestManager(mgr, g)
+	ctx, cancelFunc, wg := StartTestManager(mgr, g)
```

Comment:
Should we `defer cancelFunc()` in case there is an early exit? Same question for other tests.

Reply:
So for now I have created a context that is cancellabe in the root of the test and immeadiately after I call defer cancelFunc()